### ISSUE TITLE
Use 

### ISSUE THREAD

**Issue by GoogleCodeExporter** [NickstaDB/reflections#140]
```
Reflections currently uses the Javassist artifact from the old "javassist" 
group:

<dependency>
    <groupId>javassist</groupId>
    <artifactId>javassist</artifactId>
    <version>3.12.1.GA</version>
</dependency>

Javassist no longer publish with this group id, instead they use 
"org.javassist" :

http://search.maven.org/#search%7Cga%7C1%7Ca%3A%22javassist%22

This leads to dependency resolution failure when combining Reflections with a 
more recently updated project, like Play 2.1, which uses a current 
Javassist artifact - both jars are pulled in simultaneously as there's no way 
to tell they represent different versions of the same thing.

To fix this, please update Reflections to use this artifact:

<dependency>
    <groupId>org.javassist</groupId>
    <artifactId>javassist</artifactId>
    <version>3.17.1-GA</version>
</dependency>

What version of the product are you using?

reflections v0.9.8

http://search.maven.org/#artifactdetails%7Corg.reflections%7Creflections%7C0.9.8
%7Cjar
```

Original issue reported on code.google.com by `roberto....@gmail.com` on 22 Jan 2013 at 11:21

**Comment by GoogleCodeExporter** [comment]
```
Apologies for the poor title on this bug :( Unfortunately Google Code gives me 
no way to fix it. It should be:

"Use Javassist from 'org.javassist' group to fix dependency resolution"

```

Original comment by `roberto....@gmail.com` on 22 Jan 2013 at 11:23

**Comment by GoogleCodeExporter** [comment]
```
fixed on trunk, comments are welcomed
```

Original comment by `ronm...@gmail.com` on 20 Feb 2013 at 9:04
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
Thanks, I see this is fixed in the not-yet-unreleased v0.9.9 release of 
Reflections:

https://code.google.com/p/reflections/source/diff?spec=svn166&r=166&format=side&
path=/trunk/reflections/pom.xml


```

Original comment by `roberto....@gmail.com` on 12 Mar 2013 at 2:54
